packages/webdriverjs/tests/example/webdriverjs-example.ts

```diff
@@ -1,12 +1,23 @@
 import { assert } from 'chai';
 import { Builder, WebDriver } from 'selenium-webdriver';
+import AxeBuilder from '../../src';
```

Comment:
You can still use Lerna to link them up I think. Makes for a better example that way.